acceptance/static_creds_test.go

```diff
@@ -69,58 +68,42 @@ func (p *PluginTest) TestStaticCredsLifecycle() {
 
 			roleName := openstack.RandomString(openstack.NameDefaultSet, 4)
 
+			userId := userSetup(t, client, data, aux, userRoleName, allRoles)
```

Comment:
You can pass an exact role to the `userSetup` function, instead passing both role name and map of roles

Reply:
That's much better, thank you :)

Reply:
updated

Reply:
Or even just a role ID :smiley:  haven't noticed earlier, sorry

Reply:
It's even better

Reply:
Though there is a problem there - are you sure there will be a role with a random name you've generated? Other way return will be an empty structure and ID will always be empty:
https://go.dev/play/p/TQ5lhgdOt8R

Reply:
I'm providing a user_role here, no static one

Reply:
My bad :disappointed_relieved: